### DISCUSSION TITLE
Expand external table schema with `schema.expand()` on v5

### DISCUSSION THREAD

**Discussion by fjuniorr** [frictionlessdata/frictionlessdata.io#798] [Q&A]
In v4 I could de-reference a table schema stored in an external file and when the package descriptor was exported to a file the table schema would be expanded. Something like this:

```python
from frictionless import Package

url = 'https://raw.githubusercontent.com/transparencia-mg/datapackage-reprex/yaml-schema/datapackage.json'

package = Package(url)
package.get_resource('estados').schema.expand()
package.to_yaml('datapackage.yaml') # the schema property would be fully dereferenced here
```

In v5 I get an`AttributeError: 'Schema' object has no attribute 'expand'`. I think this is because of the [new metadata model used in v5](https://framework.frictionlessdata.io/blog/2022/08-22-frictionless-framework-v5.html#improved-model), but I was wondering if there is any way to achieve the same functionality?

**Comment by roll** [comment]
Hi @fjuniorr,

The expand command was also adding default values but if you're interested in the dialect/schema dereferencing I can quickly add something like `package.dereference()` to v5

**Reply by fjuniorr** [reply]
That would be great @roll!

Do you want me to open an issue for this?

**Reply by roll** [reply]
I'm releasing `frictionless@5.8` with this feature

**Reply by fjuniorr** [reply]
Thanks @roll !
